INSTALL.txt

```diff
@@ -12,7 +12,7 @@ Before installing Picard, you need to have these:
  * PyQt 4.5 or newer
    http://www.riverbankcomputing.co.uk/software/pyqt/download
 
- * Mutagen 1.22 or newer (1.23 or newer for AIFF support)
+ * Mutagen 1.37 or newer
```

Comment:
1.37 is required for MVNM/MVIN support only ?
Why is this exact version required exactly ?

I propose to stay on 1.22, and just add (Mutagen 1.37 or newer for MVNM/MVIN support), together with a version test in the code to enable/disable the support of those tags.